include/Dialects/LinalgTransform/TransformOpInterface.h

```diff
@@ -206,6 +207,9 @@ class TransformState {
   /// Extensions attached to the TransformState, identified by the TypeID of
   /// their type. Only one extension of any given type is allowed.
   DenseMap<TypeID, std::unique_ptr<Extension>> extensions;
+
+public:
```

Comment:
I don't see a reason for this to be public.

Reply:
right.